__tests__/app.test.js

```diff
@@ -117,3 +117,52 @@ describe("GET /reviews/:review_id", () => {
       });
   });
 });
+
+describe("GET /reviews/:review_id/comments", () => {
+  test("GET - status:200, responds with a review object", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toHaveLength(3);
+        comments.forEach((comment) => {
+          expect(comment).toEqual(
+            expect.objectContaining({

```

Comment:
We can be more specific here with what we expect back - you're making a request to a specific review_id so know what you're getting back from that review_id - use that to test against i.e. use the text from that review in the test - not just the structure, but the actual content.